### PR TITLE
GCI-4 - Patient/Visits Search Tests

### PR DESCRIPTION
**Pull request by Izaron**
This code tests search.
Comes on the page "Find Patient Record", "Active Visits", "Capture Vitals" and fill the names and IDs of patients and verifies the correctness of the results of the search. All the code is commented
Please tell me what I should do (change tests or refactor code)

Ticket: https://issues.openmrs.org/browse/GCI-4
Melange task: https://www.google-melange.com/gci/task/view/google/gci2014/5216075478204416

### CODE REVIEW

File: ui-tests/src/main/java/org/openmrs/reference/page/ActiveVisitsPage.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package org.openmrs.reference.page;
```
Comment (harsha89) [inline ui-tests/src/main/java/org/openmrs/reference/page/ActiveVisitsPage.java]: Licence header is missing!

File: ui-tests/src/main/java/org/openmrs/reference/page/FindPatientRecordPage.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package org.openmrs.reference.page;
```
Comment (harsha89) [inline ui-tests/src/main/java/org/openmrs/reference/page/FindPatientRecordPage.java]: Licence header is missing!

File: ui-tests/src/main/java/org/openmrs/reference/page/SearchPage.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package org.openmrs.reference.page;
```
Comment (harsha89) [inline ui-tests/src/main/java/org/openmrs/reference/page/SearchPage.java]: Licence header is missing!

File: ui-tests/src/test/java/org/openmrs/reference/SearchRecordTest.java [hunk 4]
```diff
@@ -0,0 +1,209 @@
+package org.openmrs.reference;
```
Comment (harsha89) [inline ui-tests/src/test/java/org/openmrs/reference/SearchRecordTest.java]: Licence header is missing!

**Comment by Izaron** [comment]
@harsha89 
Done it
There is video clip - http://youtu.be/ZFm759Lmf24

**Comment by harsha89** [comment]
Agreed there may lot's of files missing licence. But it's mandatory. :). So when you do a contribution. Please add licence as well :)

**Comment by dkayiwa** [comment]
@harsha89 and @k-joseph what do you think we should do for all the open GCI pull requests for this module?

**Comment by dkayiwa** [comment]
Closing because this is now out of synch!

**Pull request closed without merging**